torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -608,12 +614,17 @@ void ProcessGroupNCCL::ncclCommWatchdogInternal() {
         // Check for Timeouts in the WorkNCCL Operations, and abort all
         // communicators accordingly.
         if (work.timedOut()) {
-          LOG(INFO)
-              << "[Rank " << rank_
-              << "] Watchdog caught collective operation timeout for work: "
-              << work;
```

Comment:
Shouldn't we retrain the LOG message here?

Reply:
Added back the log in the workCleanupThread